ompi/tools/mpirun/main.c

```diff
@@ -23,10 +23,14 @@
 #include "opal/util/os_dirpath.h"
 #include "opal/util/os_path.h"
 #include "opal/util/path.h"
+#include "opal/util/printf.h"
 
 int main(int argc, char *argv[])
 {
-    char *evar, *pvar;
+    char *evar;
+#if OMPI_USING_INTERNAL_PRRTE
```

Comment:
Do you want `#if OMPI_USING_INTERNAL_PRRTE || OPAL_USING_INTERNAL_PMIX`?

Reply:
fixed